test/build_graph.cpp

```diff
@@ -219,3 +221,9 @@ TEST(build_graph, validate_function_for_dynamic_shape)
     EXPECT_EQ(true, make_function(true)->is_dynamic());
     EXPECT_EQ(false, make_function(false)->is_dynamic());
 }
+
+TEST(build_graph, opset1)
+{
```

Comment:
Could do `ASSERT_TRUE(is_type<op::v0::Add>(opset1::Add())`
I don't think we actually ever need to know the op version number, just the versioned op.